src/components/calcite-shell-panel/calcite-shell-panel.tsx

```diff
@@ -28,13 +33,13 @@ export class CalciteShellPanel {
   // --------------------------------------------------------------------------
 
   render() {
-    const { layout } = this;
+    const { collapsed, layout } = this;
 
-    const contentNode = (
+    const contentNode = !collapsed ? (
```

Comment:
Weren't we running into an issue in IE11 or Edge (I forget) where this pattern wasn't working properly? To fix we had to instead conditionally add an attribute and use `display:none` if I recall correctly.

Reply:
I don't recall that. I just recall not using `toggleAttribute` for edge.

Reply:
ah, I think you're right. The Edge polyfill still lets slot content in. :(